src/Grpc/test/testassets/InteropWebsite/Program.cs

```diff
@@ -37,7 +37,7 @@ public static void Main(string[] args)
             Host.CreateDefaultBuilder(args)
                 .ConfigureLogging(builder =>
                 {
-                    builder.AddConsole(o => o.DisableColors = true);
+                    builder.AddSimpleConsole(o => o.DisableColors = true);
```

Comment:
This change is probably why the gRPC interop tests failing. The tests are expecting a certain message logged to the console.

Reply:
The functionality should not have changed. I'm comparing the outputs.